app/Resources/views/newsletter/invitation_sent.html.twig

```diff
@@ -0,0 +1,23 @@
+{% extends 'base.html.twig' %}
+
+{% block banner '' %}
+{% block footer '' %}
+
+{% block page_title 'Vous êtes désormais inscrit à notre newsletter' %}
+{% block canonical_url url('newsletter_subscription') %}
+
+{% block content %}
+
+    <main class="newsletter-result">
+        <h2>Votre(vos) invitation(s) a(ont) bien été envoyée(s) !</h2>
```

Comment:
Titouan told be about a simple hardcoded page but ok, the only "easy" way I see is to use the session since this happens after a redirection, done in 61b3f8c81ef1d348421c3cc2b74d2e29dccc4cdc.